src/main/java/com/jakewharton/retrofit2/adapter/rxjava2/RxJava2CallAdapter.java

```diff
@@ -69,13 +69,13 @@
       return observable.toFlowable(BackpressureStrategy.LATEST);
     }
     if (isSingle) {
-      return observable.toSingle();
+      return observable.single(null);
```

Comment:
this does not work yet. I don't see a method of just converting it to a single without giving a default value. Will open an issue for it